### PR TITLE
Add support for Stripe 11

### PR DESCRIPTION
**Pull request by abepark01**
Add support for the [Stripe v11](https://github.com/stripe/stripe-ruby/blob/master/CHANGELOG.md#1100---2024-04-10) gem.

### CODE REVIEW

**Comment by schinery** [comment]
@rmm5t are you able to give this a look? 🙏🏻 

(Only asking as you merged the [previous Stripe bumps](https://github.com/integrallis/stripe_event/pull/163))

File: CHANGELOG.md [hunk 1]
```diff
@@ -64,107 +66,122 @@
...
-  * Fix controller test which was passing incorrectly
+
+- Improve README
+- Specify development dependency versions
+- Fix controller test which was passing incorrectly
 
 ### 0.1.0 (June 24, 2012)
-  * Initial release
+
+- Initial release
```
Comment (rmm5t) [inline CHANGELOG.md:187]: [CONCERN] I'm going to let this slide for the sake of time and the fact that it's effectively benign, but please never do this (reformatting of unrelated things) in a PR related to something else.

**Review by rmm5t — APPROVED** [review]
LGTM. 

Please take note on the comment about the abundance of changes in the changelog. For future reference, at least. Thank you.

**Comment by rmm5t** [comment]
v2.9.0 released:

https://rubygems.org/gems/stripe_event/versions/2.9.0

**Comment by schinery** [comment]
Thanks @rmm5t 🙌🏻

File: CHANGELOG.md (hunk 1 above)
Comment (abepark01) [reply CHANGELOG.md:187]: @rmm5t Thank you for the note! My editor autoformatted the text and I didn't realize it until afterward.

**Pull request merged**